### PR TITLE
【translate】doc/en/nebd_en.md

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: docs/en/nebd_en.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+[中文版](../cn/nebd.md)
+
+### Overview
+
+CURVE client serves as the entrance of the services provided by CURVE, providing dynamic link library for QEMU/CURVE-NBD for calling. As a result, restarting QEMU/CURVE-NBD is necessary when CURVE Client needs to be updated.
```
Comment (wu-hanqing) [inline docs/en/nebd_en.md:5]: 这里的for calling可以去掉

File: docs/en/nebd_en.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+</p>
+
+Figure1 shows the deployment structure of NEBD.
+
+- NEBD Client (**part1** in source code directory): NEBD Client corresponds to applications based on CURVE, including QEMU and CURVE-NBD.  An NEBD client connects to specified NEBD server through Unix Domain Socket.
+
+- NEBD Server(**part2** in source code directory)：NEBD Server is responsible for receiving the requests from part1, then call CURVE client for corresponding operations. An NEBD server can receive requests from different NEBD clients.
+
+
+Also, figure 1 shows that instead of CURVE client, NEBD client is now the component that serves the application above. In this case, the applications will still be influenced when NEBD client is being upgraded. So in our design, we simplified the processing logic of NEBD client as much as possible, which means it will only be responsible for request forwarding and limited retries for requests if needed.
```
Comment (wu-hanqing) [inline docs/en/nebd_en.md:23]: the processing logic of NEBD -> the business logic of NEBD

File: docs/en/nebd_en.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+There are few steps for NEBD server/CURVE client's upgrade:
+
+1. Install the latest version of CURVE client/NEBD server
+2. Stop running processes of part2
+3. Restart processes of part2
+
+
+In our practice, we use daemon to monitor the processes of part2, and start them if not exist.
+
+Also, notice that from the stop of processes of part2 to the start of the new ones, only 1 to 5 seconds are required in our test and production environment.
```
Comment (wu-hanqing) [inline docs/en/nebd_en.md:34]: notice that from the stop of processes of part2 to the start of the new ones，这里`processes`和最后的`ones`，是不是不用s？

File: docs/en/nebd_en.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- File Manager: Manage opened files on NEBD server.
+
+- IO Executor: Responsible for the actual execution of file requests, which calls the interface of CURVE client and send requests to storage clusters.
+
+- Metafile Manager: Manage metadata files, and also responsible for metadata persistence, or load persistent data from files.
+
+
+### Key Points
+
+1. Retry procedure of part1:
```
Comment (wu-hanqing) [inline docs/en/nebd_en.md:71]: Retry policy of part1

File: docs/en/nebd_en.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+   As what we've mentioned above, part1 only execute limited retries, and this characteristic can be reflected in two aspects:
+
+   1. There's no time out for RPC requests from part1.
+
+   2. Part1 only executes retries for errors of RPC requests themselves, and forward error      codes return by RPC to upper level directly.
```
Comment (wu-hanqing) [inline docs/en/nebd_en.md:77]: error和codes之间的空格
error codes return by RPC -> error codes return***ed*** by RPC？

File: docs/en/nebd_en.md (hunk 1 above)
Comment (SeanHai) [reply docs/en/nebd_en.md:5]: > 这里的for calling可以去掉

done

File: docs/en/nebd_en.md (hunk 2 above)
Comment (SeanHai) [reply docs/en/nebd_en.md:23]: > the processing logic of NEBD -> the business logic of NEBD

done

File: docs/en/nebd_en.md (hunk 3 above)
Comment (SeanHai) [reply docs/en/nebd_en.md:34]: > notice that from the stop of processes of part2 to the start of the new ones，这里`processes`和最后的`ones`，是不是不用s？

done

File: docs/en/nebd_en.md (hunk 4 above)
Comment (SeanHai) [reply docs/en/nebd_en.md:71]: > Retry policy of part1

done

File: docs/en/nebd_en.md (hunk 5 above)
Comment (SeanHai) [reply docs/en/nebd_en.md:77]: > error和codes之间的空格
> error codes return by RPC -> error codes return_**ed**_ by RPC？

done

**Review by wu-hanqing — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**